src/libraries/System.Security.Cryptography.Cose/tests/CoseHeaderMapTests.cs

```diff
@@ -0,0 +1,248 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using System.Formats.Cbor;
+using System.Collections.Generic;
+using static System.Security.Cryptography.Cose.Tests.CoseTestHelpers;
+
+namespace System.Security.Cryptography.Cose.Tests
+{
+
+    public class CoseHeaderMapTests
+    {
+        [Fact]
+        public void SetValue_GetValue_KnownHeader()
+        {
+            var alg = new CoseHeaderLabel(KnownHeaders.Alg);
+            var contentType = new CoseHeaderLabel(KnownHeaders.ContentType);
+            var kid = new CoseHeaderLabel(KnownHeaders.Kid);
+            var iV = new CoseHeaderLabel(KnownHeaders.IV);
+            var partialIV = new CoseHeaderLabel(KnownHeaders.PartialIV);
```

Comment:
If we want to declare them using named values, it seems like they should just be using `CoseHeaderLabel.Algorithm` and friends.  If we're trying to ensure that those published values have the correct backing value, then they should be using literals.

Reusing a const from the source project just means that the test will always look right, instead of checking that the source matches the specified behavior.